openerp/models.py

```diff
@@ -2434,7 +2434,7 @@ def _auto_init(self, cr, context=None):
 
         store_compute = False
         stored_fields = []              # new-style stored fields with compute
-        todo_end = []
+        todo_end = context['todo']
```

Comment:
Why using a mutable from the context?
What's wrong with current solution of returning the values?